backend-peer/src/lib/services/cloudagent.ts

```diff
@@ -72,7 +81,39 @@ export class CloudagentManager {
     return agentInfo as AgentInfo
   }
 
-  receiveImplicitInvitation = async (did: string) => {
+  receiveImplicitInvitation = async (
+    did: string,
+    waitUntilCompleted: boolean = false
+  ) => {
+    let connectionId: string
```

Comment:
doesn't look like `connectionId` has its value set anywhere?

Reply:
This should have been caught in the e2e tests - can we define a test case for this?